src/sidebar/components/AutocompleteList.js

```diff
@@ -85,20 +85,23 @@ export default function AutocompleteList({
   const isHidden = list.length === 0 || !open;
   return (
     <div className="relative">
-      <Card
-        classes={classnames(
+      <div
```

Comment:
Just adding an extra `div` here to take care of positioning concerns instead of smashing that into `Card`. We don't need to "override full-width" anymore as `Card` now has some (d'oh! Undocumented https://github.com/hypothesis/frontend-shared/issues/648 ) `width` controls.